ironic-config/httpd.conf.j2

```diff
@@ -0,0 +1,82 @@
+ServerRoot "/etc/httpd"
+{%- if env.LISTEN_ALL_INTERFACES %}
+Listen [::]:{{ env.HTTP_PORT }}
+{% else %}
+Listen {{ env.IRONIC_URL_HOST }}:{{ env.HTTP_PORT }}
+{% endif %}
+Include conf.modules.d/*.conf
+User apache
+Group apache
+
+ServerAdmin root@localhost
+ServerName www.example.com:80
```

Comment:
I don't think we should be doing this? Maybe use IRONIC_IP and the actual port here?